.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
+* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it
+      should be force landed and forced-landing tag is set.
+* [ ] Commit messages follows the guidelines outlined
+      [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
+* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.
+
+### Gatekeeper:
+
+* [ ] You are the appropriate gatekeeper to be landing the patch.
+* [ ] The PR has 2 reviews by people familiar with the code.
+* [ ] Any appropriate watchers have had a chance to review the PR.
+* [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested
+      changes.
+* [ ] Githooks were used. If not, there is a sufficient reason to move forward.  Check copyrights
+      if githooks were not used.
+* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future
+      PRs.
+* [ ] All builds have passed.  Check non-required builds to ensure they don't indicate any problem
+      such as a compiler warning on extraneous platforms.
```

Comment:
This is step 2, PR is Green isn't it?  All compiler steps should be marked as required except the landings-builds ones which only trigger if you modify certain files.